tests/xcm-simulator/src/tests/experimental.rs

```diff
@@ -252,14 +252,14 @@ fn test_async_xcm_contract_call_no_ce() {
         /// Max fee for whole XCM operation in foreign chain
         /// This includes fees for sending XCM back to original
         /// chain via Transact(pallet_xcm::send).
-        pub foreign_base_fee: MultiAsset,
+        pub foreign_base_fee: xcm::v3::MultiAsset, // match smart-contract xcm version
```

Comment:
`pallet-contracts` still has v3 hardcoded?

Reply:
no, smart-contract we using for testing is build with v3 https://github.com/AstarNetwork/ink-test-contracts/blob/e4878fba2cb5f92c5a965f46bc542f1b128d75a3/contracts/async-xcm-call-no-ce/lib.rs#L144

Reply:
Uh, cannot wait until it's deprecated 😂